src/utils/LinearAlg2D.cpp

```diff
@@ -18,6 +18,20 @@ float LinearAlg2D::getAngleLeft(const Point& a, const Point& b, const Point& c)
     const Point bc = c - b;
     const coord_t dott = dot(ba, bc); // dot product
     const coord_t det = ba.X * bc.Y - ba.Y * bc.X; // determinant
+    if (det == 0)
+    {
+        if (
+            (ba.X != 0 && (ba.X > 0) == (bc.X > 0))
+            || (ba.X == 0 && (ba.Y > 0) == (bc.Y > 0))
+            }
```

Comment:
This bracket should be }, not ).

Reply:
lol. I used the Github interface for this fix.

Reply:
I've found out how to change code in a PR from within Github. I'm somewhat of a master-hacker, you know. :P